corgi/tasks/brew.py

```diff
@@ -17,18 +17,28 @@
     ProductStream,
     SoftwareBuild,
 )
-from corgi.tasks.common import RETRY_KWARGS, RETRYABLE_ERRORS, get_last_success_for_task
+from corgi.tasks.common import (
+    RETRY_KWARGS,
+    RETRYABLE_ERRORS,
+    create_relations,
+    get_last_success_for_task,
+)
 from corgi.tasks.errata_tool import slow_load_errata
 from corgi.tasks.sca import cpu_software_composition_analysis
 
 logger = get_task_logger(__name__)
 
+build_type = SoftwareBuild.Type.KOJI if settings.COMMUNITY_MODE_ENABLED else SoftwareBuild.Type.BREW
```

Comment:
This is minor, but I think it might be better to use uppercase here so it's clear this is a constant. That also lets you use build_type below instead of b_type, to be consistent with / match build_id:
```suggestion
BUILD_TYPE = SoftwareBuild.Type.KOJI if settings.COMMUNITY_MODE_ENABLED else SoftwareBuild.Type.BREW
```